.travis.yml

```diff
@@ -3,13 +3,10 @@ language: php
 dist: trusty
```

Comment:
It's good time to update distro to bionic https://docs.travis-ci.com/user/reference/bionic/#php-support